pkg/registry/apis/query/query.go

```diff
@@ -63,6 +60,7 @@ func (b *QueryAPIBuilder) doQuery(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	w.Header().Set("Content-Type", "application/json")
```

Comment:
^^ needs set explicitly

Reply:
is error response also a valid json or should it have a different content type on errors?